src/main/scala/nl/knaw/dans/easy/multiDeposit/actions/CreateSpringfieldAction.scala

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright (C) 2015-2016 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.multiDeposit.actions
+
+import java.io.File
+
+import nl.knaw.dans.easy.multiDeposit.{Dataset, _}
+import nl.knaw.dans.easy.multiDeposit.actions.CreateSpringfieldAction._
+import org.slf4j.LoggerFactory
+
+import scala.util.{Failure, Success, Try}
+import scala.xml.PrettyPrinter
+
+case class Video(name: String, fileSip: Option[String], subtitles: Option[String])
+
+case class CreateSpringfieldAction(row: Int, datasets: Datasets)(implicit settings: Settings) extends Action(row) {
```

Comment:
Should be plural: `CreateSpringfieldActions`. The "Actions" here is the last part of the generated xml file `springfield-actions.xml` and does not refer to the fact that this class extends `Action`. I notice that the new `Action` subclasses you created have names ending in `Action`. Probably best remove that suffix. From the names being verb-object pairs it should be obvious that they encapsulate actions.